src/rospkg/os_detect.py

```diff
@@ -146,7 +146,10 @@ def __init__(self, lsb_name, get_version_fn=None):
             self.lsb_info = None
 
     def is_os(self):
-        return self.lsb_info is not None and self.lsb_info[0] == self.lsb_name
+        # Work around platform returning 'Ubuntu' and distro returning 'ubuntu'
+        name_is_same = self.lsb_info[0].lower() == self.lsb_name.lower()
```

Comment:
We should check if `self.lsb_info is None` first to avoid a runtime error.

Reply:
Checking `None` first in 128748fe5329493e961bcc3f2f59ff59b485c331